.github/workflows/main.yaml

```diff
@@ -0,0 +1,92 @@
+name: Main
+on: [push, pull_request]
+jobs:
```

Comment:
There was a recent discussion on this and our plan was to go the way of publishing on `release/` branches: https://github.com/hobbyfarm/ui/pull/57#issuecomment-594244509

I'd be interested to hear if you prefer a different approach.

Reply:
Branching model aside, this will push any Git Tag to Dockerhub.  You can make a release branch to do final prep there if you want, just tag it when you're ready to go.

It could be smarter about what it tags as `latest` for example process a semver and make sure it is not a prerelease tag.

Reply:
I think the "smarter" logic could come later though, this is at least feature parity with how it works now